### PR TITLE
Check template dir for language file

### PR DESCRIPTION
**Pull request by scottcwilson**
These files are pulling in language files without using `zen_get_file_directory()`, which can lead to inconsistent results.

### CODE REVIEW

File: includes/classes/ajax/zcAjaxPayment.php [hunk 1]
```diff
@@ -180,7 +180,8 @@ public function prepareConfirmation()
     }
 
     $current_page_base = FILENAME_CHECKOUT_CONFIRMATION;
-    require_once (DIR_WS_LANGUAGES.$_SESSION['language'].'.php');
+    require_once(zen_get_file_directory(DIR_FS_CATALOG . DIR_WS_LANGUAGES
+. "/", $_SESSION['language'].'.php', 'false'));
```
Comment (drbyte) [inline includes/classes/ajax/zcAjaxPayment.php]: Let's put this on one line.
Why did you add the `. "/"`?

File: includes/init_includes/init_templates.php [hunk 2]
```diff
@@ -52,16 +52,8 @@
...
-    $template_dir_select = $template_dir . '/';
-  /**
-   * include the template language overrides
-   */
-      include_once(DIR_WS_LANGUAGES . $template_dir_select . $_SESSION['language'] . '.php');
-  } else {
-    $template_dir_select = '';
-      //  include_once(DIR_WS_LANGUAGES . $template_dir_select . $_SESSION['language'] . '.php');
-  }
+  include_once(zen_get_file_directory(DIR_FS_CATALOG . DIR_WS_LANGUAGES . "/", $_SESSION['language'].'.php', 'false'));
```
Comment (drbyte) [inline includes/init_includes/init_templates.php]: Why did you add the `. "/"`?

File: includes/languages/english.php [hunk 3]
```diff
@@ -632,12 +632,7 @@ function zen_date_raw($date, $reverse = false) {
   $file_list = array(FILENAME_EMAIL_EXTRAS, FILENAME_HEADER, FILENAME_BUTTON_NAMES, FILENAME_ICON_NAMES, FILENAME_OTHER_IMAGES_NAMES, FILENAME_CREDIT_CARDS, FILENAME_WHOS_ONLINE, FILENAME_META_TAGS); 
   foreach ($file_list as $file) { 
     $file = str_replace(".php","",$file); 
-    if (file_exists(DIR_WS_LANGUAGES . $_SESSION['language'] . '/' . $template_dir . '/' . $file . '.php')) {
-      $template_dir_select = $template_dir . '/';
-    } else {
-      $template_dir_select = '';
-    }
-    require_once DIR_WS_LANGUAGES . $_SESSION['language'] . '/' . $template_dir_select . $file . '.php'; 
+    require_once(zen_get_file_directory(DIR_FS_CATALOG . DIR_WS_LANGUAGES . $_SESSION['language'] . "/", $file . ".php", 'false'));
```
Comment (drbyte) [inline includes/languages/english.php]: Why did you add the `. "/"`?  And why the double-quotes around `".php"`?

Comment (scottcwilson) [reply includes/languages/english.php]: You do need the '/' because $_SESSION['language'] doesn't have a trailing slash.  This is different from your finding above, which was a bug.

File: includes/init_includes/init_templates.php (hunk 2 above)
Comment (scottcwilson) [reply includes/init_includes/init_templates.php]: fixed.
